src/annotations/validateAnnotations/AdulthoodValidator.java

```diff
@@ -0,0 +1,31 @@
+package annotations.validateAnnotations;
+
+import annotations.Adulthood;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.time.LocalDate;
+
+
+public class AdulthoodValidator implements Validator {
+
+    @Override
+    public void fieldValidator(Class<?> className) {
```

Comment:
Make this class Generic and use generic type as a method parameter 
public class AdulthoodValidator <T> implements Validator {
@Override
    public void fieldValidator(T dto) {